repo/packages/H/hdfs/3/resource.json

```diff
@@ -0,0 +1,13 @@
+{
+  "images": {
+    "icon-small": "https://downloads.mesosphere.com/hdfs/assets/icon-service-hdfs-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/hdfs/assets/icon-service-hdfs-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/hdfs/assets/icon-service-hdfs-large.png"
+  },
+  "assets": {
+    "uris": {
+      "hdfs-mesos-scheduler": "https://downloads.mesosphere.com/hdfs/hdfs-mesos-0.1.9.tgz",
```

Comment:
this lines needs to be `https://downloads.mesosphere.com/hdfs/assets/hdfs-mesos-0.1.9.tgz`